ansible/roles/postgresql/templates/pgbouncer.ini.j2

```diff
@@ -9,7 +9,7 @@
 [databases]
 {% for db in postgresql_dbs %}
 {% set db_host = db.get('host', localsettings.PG_DATABASE_HOST) %}
-{% if db_host == inventory_hostname or db_host == hot_standby_master or is_monolith|bool %}
+{% if db_host == inventory_hostname or (hot_standby_master is defined and db_host == hot_standby_master) or is_monolith|bool %}
```

Comment:
this could also be 
`{% if db.get('host', localsettings.PG_DATABASE_HOST) == inventory_hostname or db.get('host', localsettings.PG_DATABASE_HOST) == hot_standby_master|default(None) or is_monolith|bool %}`
which we use in `pg_hba.conf` and is a little more concise